docs/DxcOnLinux.rst

```diff
@@ -0,0 +1,78 @@
+=======================================
+DirectXShaderCompiler on Linux & Mac OS
+=======================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+============
+
+Due to several `requests <https://github.com/Microsoft/DirectXShaderCompiler/issues/1236>`_
+from the community, we have started the effort to enable compilation and running
+of DirectXShaderCompiler on non-Windows platforms (Linux and Mac OS).
+
+
+Building DirectXShaderCompiler
+==============================
+You can follow these steps to build DirectXShaderCompiler on Linux/MacOS:
+
+(We are assuming you have cloned the repository to `~/DirectXShaderCompiler`)
```

Comment:
We can use `<dxc-src-dir>` and `<dxc-build-dir>` so that we don't need to have this explanation.

Reply:
Done